### PR TITLE
Adding support for custom food classifier

### PR DESCRIPTION
**Pull request by KSlachta**

### CODE REVIEW

**Comment by germanattanasio** [comment]
@KSlachta Make sure you test this with different configurations so that you don't get surprises during the release

**Comment by KSlachta** [comment]
@germanattanasio What kind of configurations are you referring to?

**Comment by germanattanasio** [comment]
@KSlachta testing with different images

File: public/js/classresults.jsx [hunk 1]
```diff
@@ -373,6 +373,27 @@ class CustomClassifyScoreTable extends React.Component {
...
 
+class FoodScoreTable extends React.Component {
+  render() {
+    return (
+        <div className="results-table--container">
+          <JsonLink rawjson={this.props.rawjson}/>
+          <table className="base--table results-table">
+            <ScoreTableHeader title={this.props.category}/>
+            <tbody className="base--tbody">
+            {this.props.items.map(function(item) {
```
Comment (germanattanasio) [inline public/js/classresults.jsx]: You can write this as 
```jsx
{this.props.items.map(item =>(
  <ClassifyScoreRow key={item.class} name={item.class} score={item.score.toFixed(2)} />
 ))}
```

File: public/js/classresults.jsx [hunk 2]
```diff
@@ -468,11 +490,25 @@ export function classifyScoreTable(results, tagid) {
 
   let target = typeof(tagid) === 'string' ? document.getElementById(tagid) : tagid;
   let tags = jpath.jpath('/images/0/classifiers/0/classes',results,[]);
+  let food = jpath.jpath('/images/0/classifiers/1/classes',results,[]);
+  var tagsJ =  jpath.jpath('/images/0/classifiers/0', results) // Json for tags
```
Comment (germanattanasio) [inline public/js/classresults.jsx]: use `let`

**Pull request merged**